tools/addlicense/addlicense_test.go

```diff
@@ -0,0 +1,204 @@
+package main
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func Test_copyright_re(t *testing.T) {
```

Comment:
Rename test in accordance with https://ieftimov.com/posts/testing-in-go-naming-conventions/